ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -30,8 +30,11 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
   }
 
   private def isShowCreation(): Boolean = {
-    // TODO: write the logic to compute the correct answer here
-    true
+    val numberOfNewRecipes: Long = db.countRecipesInGivenStatus(New)
```

Comment:
Please note that every time we call isShowCreation (i.e every time a user wants to participate) we are making 3 requests to the DB just to decide what view to show them. This is not a problem at the moment, nor may it ever be with the kind of load we are likely to expect but this is something to just be mindful of. No changes are required.